l10n/override/zh_CN/_layouts/extensions.html

```diff
@@ -0,0 +1,60 @@
+---
+layout: base
+---
+
+<div class="grid-wrapper extensions-page">
+  <div class="grid__item width-12-12">
+    <h1 class="text-caps">{{ page.title }}</h1>
+  </div>
+  <div class="grid__item width-4-12 width-12-12-m toc">
+    <h2>Categories</h2>
+    <ul>
+    {% for item in site.data.extensions.categories %}
+      <li><a href="#{{ item.cat-id }}">{{ item.category }}</a></li>
+    {% endfor %}
+    </ul>
+  </div>
+  <div class="grid__item width-8-12 width-12-12-m">
+<div markdown="1">
+<p>
+Think of Quarkus extensions as your project dependencies.
+Extensions configure, boot and integrate a framework or technology into your Quarkus application.
+They also do all of the heavy lifting of providing the right information to GraalVM for your application to compile natively.
+</p>
```

Comment:
I think these words could be better to translate to Chinsese.
```
可以把Quarkus扩展当成是您的项目依赖。把扩展配置、启动框架或技术集成到您的 Quarkus 应用程序中。
同样还完成了向 GraalVM 提供正确信息以供您的应用程序进行本地编译的所有繁琐工作。
```

Reply:
Done.